About.html

```diff
@@ -0,0 +1,27 @@
+<html>
+<head>
+	<title>About Me</title>
+</head>
+<body>
+	<br>
+<a href="index.html">Home</a>
+
+<h1> We will connect you with a Dog Match Momentarily</h1>
+
+
+
+<img src = "pets.jpg" alt = "Picture of a happy dog" height="400" width="400">
+<br>
+
+<p> What makes this website so good? It has Div element which seperates the code block, but most importantly this site is so advance because they included functionality, and used javascript for it's framework. Also, I see that they have an API for Facebook, which allows user to easily log in by signing in with their social media account.</p> <a href= "https://barkbuddy.com/">About Bark Buddy</a>
+<H3>Benefits of DogTinder<h3>
+
+<ul style="list-style-type:disc">
+  <li>Easy to Meet New Dogs in the area in real time</li>
+  <li>Great way to adopt and sell Pet</li>
+  <li>Happy Dog</li>
+</ul>  
+</ul>
+
+</body>
+</html>
```

Comment:
In terms of HTML styling, there should be a new line at the end of the file. It's a setting you can put in Sublime as well that will do it automatically for your files.